GiTinder.Tests/Controllers/LoginRequestBodyFactory.cs

```diff
@@ -16,6 +16,15 @@ public static LoginRequestBody CreateLoginRequestBodyWithNullUsername()
             };
         }
 
+        public static LoginRequestBody CreateLogoutRequestBodyWithNullUsertoken()
+        {
+            return new LoginRequestBody()
+            {
+                Username = "",
+                AccessToken = "X-Gitinder-Token"
+            };
+        }
```

Comment:
@Riceqrisp remove this method after checking if its used

Reply:
removed .